src/cli/list.rs

```diff
@@ -77,13 +77,34 @@ fn serde_skip_is_editable(editable: &bool) -> bool {
     !(*editable)
 }
 
+#[derive(Serialize, PartialEq, Eq, PartialOrd, Ord)]
+#[serde(rename_all = "lowercase")]
+enum KindPackage {
+    Conda,
+    Pypi,
+}
+impl KindPackage {
+    fn color(&self) -> Color {
+        match self {
+            KindPackage::Conda => Color::Green,
+            KindPackage::Pypi => Color::Blue,
+        }
+    }
+    fn fancy_display(&self) -> console::StyledObject<&str> {
+        match self {
+            KindPackage::Conda => console::style("conda").fg(self.color()).bold(),
```

Comment:
maybe we could add a constant `Style` object ( like PLATFORM_STYLE ) so we could reuse it later